haskell-language-server.cabal

```diff
@@ -2179,119 +2243,140 @@ test-suite ghcide-tests
 
 flag ghcide-bench
   description: Build the ghcide-bench executable
-  default:     True
+  default: True
 
 executable ghcide-bench
-    import: defaults
-    if !flag(ghcide-bench)
-      buildable: False
-    build-depends:
-        aeson,
-        bytestring,
-        containers,
-        data-default,
-        extra,
-        filepath,
-        hls-plugin-api,
-        hls-test-utils,
-        lens,
-        lsp-test,
-        lsp-types,
-        optparse-applicative,
-        process,
-        safe-exceptions,
-        hls-graph,
-        shake,
-        tasty-hunit >= 0.10,
-        text,
-        haskell-language-server:ghcide-bench-lib,
-    hs-source-dirs: ghcide-bench/exe
-    ghc-options: -threaded -Wall -Wno-name-shadowing -rtsopts
-    main-is: Main.hs
-    default-extensions:
-        LambdaCase
-        OverloadedStrings
-        RecordWildCards
-        ViewPatterns
+  import: defaults
+
+  if !flag(ghcide-bench)
+    buildable: False
+  build-depends:
+    aeson,
+    bytestring,
+    containers,
+    data-default,
+    extra,
+    filepath,
+    haskell-language-server:ghcide-bench-lib,
+    hls-graph,
+    hls-plugin-api,
+    hls-test-utils,
+    lens,
+    lsp-test,
+    lsp-types,
+    optparse-applicative,
+    process,
+    safe-exceptions,
+    shake,
+    tasty-hunit >=0.10,
+    text,
+
+  hs-source-dirs: ghcide-bench/exe
+  ghc-options:
+    -threaded
+    -Wall
+    -Wno-name-shadowing
+    -rtsopts
+
+  main-is: Main.hs
+  default-extensions:
+    LambdaCase
+    OverloadedStrings
+    RecordWildCards
+    ViewPatterns
 
 library ghcide-bench-lib
-    import: defaults
-    hs-source-dirs: ghcide-bench/src
-    ghc-options: -Wall -Wno-name-shadowing
-    exposed-modules:
-        Experiments.Types
-        Experiments
-    build-depends:
-        aeson,
-        async,
-        binary,
-        bytestring,
-        deepseq,
-        directory,
-        extra,
-        filepath,
-        ghcide:{ghcide},
-        hashable,
-        lens,
-        lsp-test,
-        lsp-types,
-        optparse-applicative,
-        parser-combinators,
-        process,
-        safe-exceptions,
-        shake,
-        text,
-        hls-test-utils,
-        row-types
-    default-extensions:
-        LambdaCase
-        RecordWildCards
-        ViewPatterns
+  import: defaults
+  hs-source-dirs: ghcide-bench/src
+  ghc-options:
+    -Wall
+    -Wno-name-shadowing
 
+  exposed-modules:
+    Experiments
+    Experiments.Types
+
+  build-depends:
+    aeson,
+    async,
+    binary,
+    bytestring,
+    deepseq,
+    directory,
+    extra,
+    filepath,
+    ghcide,
```

Comment:
In this case I don't think it changes the semantics, but starting with version 1.5.0.3, Gild would keep this as `ghcide:{ghcide}`. See https://github.com/tfausak/cabal-gild/pull/115.